openstack/cronus/templates/nebula/_config.yaml.tpl

```diff
@@ -180,6 +180,16 @@ nebula:
   {{- end }}
 {{- end }}
 {{- end }}
-
+{{- if .Values.nebula.cors }}
```

Comment:
I see you just changed :) another option would have been to update the ci/test-values.yaml. Not sure why the CI doesn't use the charts values files 🤔

Reply:
Ah yea. I wondered why my values entries didn't work. :-)